cluster-state-service/internal/features/wrappers/css_wrapper.go

```diff
@@ -102,19 +102,17 @@ func (wrapper CSSWrapper) TryListTasksWithInvalidCluster(cluster string) (string
 	return "", "", errors.New("Expected an exception when calling ListTasks with invalid cluster, but none received")
 }
 
-func (wrapper CSSWrapper) TryListTasksWithAllFilters(status string, cluster string, startedBy string) (string, string, error) {
+func (wrapper CSSWrapper) TryListTasksWithAllFilters(status string, cluster string, startedBy string) ([]*models.Task, error) {
```

Comment:
Let's rename this to `ListTasksWithAllFilters` since this isn't returning an error anymore